src/ert/_c_wrappers/job_queue/function_ert_script.py

```diff
@@ -7,46 +7,33 @@
 class FunctionErtScript(ErtScript):
     def __init__(self, ert, function_name, argument_types, argument_count):
         super().__init__(ert)
+        self._arg_types = argument_types
+        self._arg_count = argument_count
+        self._func_name = function_name
 
-        parsed_argument_types = []
-
+    def run(self, *args):
+        ert = self.ert()
         if ert is not None:
-            self.__function = ResPrototype(f"void* {function_name}(void*, stringlist)")
+            str_args = StringList()
+            for arg in args:
+                str_args.append(arg)
+            pointer = ert.from_param(ert) if hasattr(ert, "from_param") else ert
```

Comment:
I hope this functionality is not in use, so could we add a log-statement to see if that is the case?

Reply:
I also hope this functionality is not in use, can you point me to an example of logging similar to what you would like to have logged here?

Reply:
Dropped changes to function_ert_script they were not needed